packages/beacon-node/src/chain/produceBlock/produceBlockBody.ts

```diff
@@ -653,7 +664,7 @@ export async function produceCommonBlockBody<T extends BlockType>(
   }
 
   const endSyncAggregate = stepsMetrics?.startTimer();
-  if (blockEpoch >= this.config.ALTAIR_FORK_EPOCH) {
+  if (ForkSeq[fork] >= ForkSeq.altair) {
```

Comment:
Required so we properly call this post altair, also using the epoch is not something we do a lot, the change shouldn't make a difference and we saw a bit not having to calculate block epoch